### PR TITLE
New PR, Readded Chip8AllInOne.ch8

### PR DESCRIPTION
**Pull request by GamingMadster**
Still no clue why those hashes got all screwed.
Hopefully all is good now :D

### CODE REVIEW

**Comment by GamingMadster** [comment]
...and apparently it says I changed 14k lines of the JSON. Neat.

**Comment by Estus-Dev** [comment]
> ...and apparently it says I changed 14k lines of the JSON. Neat.

Well it looks like you added 11 and modified 7,282. They just show up as added and removed when you change them.

Without pulling it down to verify, it looks like it's line endings. Try setting your editor to use `\n` (Unix-style) endings rather than `\r\n` (Windows-style) and reformat the file. Also set git to convert `\r\n` line endings to `\n` when committing, most projects will expect that.

**Comment by GamingMadster** [comment]
> > ...and apparently it says I changed 14k lines of the JSON. Neat.
> 
> Well it looks like you added 11 and modified 7,282. They just show up as added and removed when you change them.
> 
> Without pulling it down to verify, it looks like it's line endings. Try setting your editor to use `\n` (Unix-style) endings rather than `\r\n` (Windows-style) and reformat the file. Also set git to convert `\r\n` line endings to `\n` when committing, most projects will expect that.

Pretty sure I fixed it, or well, I hope so anyway.

File: database/programs.json [hunk 1]
```diff
@@ -3,53 +3,53 @@
     "title": "15 Puzzle",
     "description": "15 Puzzle, by Roger Ivie\n\nRearrange the tiles into the sequence 0-F. Moves tile by it associated key.",
     "release": "19xx",
-    "authors": ["Roger Ivie"],
+    "authors": [ "Roger Ivie" ],
```
Comment (Estus-Dev) [inline database/programs.json]: Looks like Visual Studio is also formatting with spaces around strings. Can you close the file in visual studio and rerun `npm run format` please, just so there's not quite so much noise in the diff?

Comment (GamingMadster) [reply database/programs.json]: Just did so, now it's saying I added 8000+ lines.

¯\\_(ツ)_/¯

Comment (Estus-Dev) [reply database/programs.json]: Yeah. `/database/programs.json` is still wrong, but there's a new `/programs.json` that looks correct. Just move that into the `/database/` folder and you should be good.

Comment (GamingMadster) [reply database/programs.json]: Alright, I've done that. Should be all good to go.

**Comment by Timendus** [comment]
Thanks for your contribution Madster! Hope the next one goes more smoothly :smile:

**Pull request merged**